src/sentry/static/sentry/app/views/performance/styles.tsx

```diff
@@ -10,43 +10,31 @@ export const GridCellNumber = styled(GridCell)`
   text-align: right;
 `;
 
-export const SummaryGridRow = styled('tr')`
-  display: grid;
-  grid-template-columns: auto minmax(70px, 170px) minmax(70px, 230px);
-`;
-
-export const HeaderContainer = styled('div')`
+export const DoubleHeaderContainer = styled('div')`
   display: grid;
   grid-template-columns: 1fr 1fr;
-  padding: ${space(2)} ${space(1.5)};
+  padding: ${space(2)} ${space(3)} ${space(1)} ${space(3)};
+  grid-gap: ${space(3)};
 `;
 
-export const HeaderTitle = styled('h3')`
+export const HeaderTitle = styled('div')`
```

Comment:
Shouldn't headings be h* elements?

Reply:
I've been using the theme.tsx file for font sizes so I think they override the h*s no matter what.